src/style.css

```
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+  line-height: 22px;
+
+}

```

also now you just import these fonts, you need to use the with font family

```suggestion
@font-face {
  font-family: Avenir-Book;
  src: url("./fonts/Avenir-Book.ttf");
  font-weight: 300;
}

@font-face {
  font-family: Avenir-Heavy;
  src: url("./fonts/Avenir-Heavy.ttf");
}
```